src/main/webapp/js/chart-loader.js

```diff
@@ -0,0 +1,31 @@
+
+google.charts.load("current", {packages: ["corechart"],});
+google.charts.setOnLoadCallback(
+  function() {
```

Comment:
It isn't necessary to add the extra layer of `function()` here. I would just reference the function by name.

That is:

    google.charts.setOnLoadCallback(callbackName);

Not:

    google.charts.setOnLoadCallback(function() {
      callbackName();
    });

There is no difference between the two statements logically, and the first one is simpler.

Reply:
Currently the implementation doesn't make sense when I am only using one chart, but when using multiple charts, I think that this implementation is better.
```
google.charts.setOnLoadCallback(function() {
  callbackName1();
  callbackName2();
  callbackName3();
});
```
than this:
```
google.charts.setOnLoadCallback(callbackName1);
google.charts.setOnLoadCallback(callbackName2);
google.charts.setOnLoadCallback(callbackName3);
```

Reply:
That's fine, though why not:

     google.charts.setOnLoadCallback(chartsLibraryLoaded);

With:

     var chartsLibraryLoaded = function() {
       callback1();
       callback2();
       callback3();
     };

I don't see why it has to be an anonymous inline function. That being said, this is minor stylistic point, and either approach is valid.